### ISSUE TITLE
TypeError: writePaperHeader() takes exactly 4 arguments (3 given)

### ISSUE THREAD

**Issue by artillect** [aspiers/ly2video#69]
OS: Manjaro Linux 17.1.0
Lilypond: 2.18.2

After installing ly2video, I tried to use it to convert one of my ly files to a video, and I got this log. I might be using it incorrectly, but I am pretty sure I'm using it properly.

```
$ ly2video -i Riley-Groeschel-The-Carpenter-s-Daughter.ly 
LilyPond was found.
FFmpeg was found.
TiMidity++ was found.
------------------------------------------------------------
Version in Riley-Groeschel-The-Carpenter-s-Daughter.ly: unspecified
------------------------------------------------------------
Generating PNG and MIDI files ...
------------------------------------------------------------
GNU LilyPond 2.18.2
Processing `/home/riley/Documents/ly2video.tmp/unconverted.ly'
Parsing...
Interpreting music...
warning: forced break was overridden by some other event, should you be using bar checks?
[8][16]
Preprocessing graphical objects...
Finding the ideal number of pages...
Fitting music on 1 page...
Drawing systems...
Layout output to `unconverted.preview.eps'...
Converting to PNG...
Success: compilation successfully completed
------------------------------------------------------------
Generated PNG and MIDI files
Looking for staff lines in unconverted.preview.png
First staff line found at (0, 76)
Found 5 staff lines
Traceback (most recent call last):
  File "/usr/bin/ly2video", line 11, in <module>
    load_entry_point('ly2video==0.4.1', 'console_scripts', 'ly2video')()
  File "/usr/lib/python2.7/site-packages/ly2video/cli.py", line 1535, in main
    numStaffLines, titleText, lilypondVersion)
  File "/usr/lib/python2.7/site-packages/ly2video/cli.py", line 1476, in sanitiseLy
    numStaffLines, lilypondVersion)
TypeError: writePaperHeader() takes exactly 4 arguments (3 given)
```

**Comment by mikeblackstock** [comment]
I was getting that same error. As  near as I can tell, it happens when there is no \version number in
the lilypond input file.

The relevant code in the cly.py file of ly2video:
1473:    # if I didn't find \version, write own paper block 
1474:   if not paperBlock:
1475:        leftPaperMarginPx = writePaperHeader(fSanitisedLyFile,
1476:                                            numStaffLines, lilypondVersion)

The call to writePaperHeader is missing a parameter, the dpi parameter,
required by the function def at line 386:

386: def writePaperHeader(fFile, dpi, numOfLines, lilypondVersion):

**Comment by aspiers** [comment]
Fixed by #71.
